tests/typecheck/fields/test_related.yml

```
@@ -1114,6 +1115,13 @@
 
         reveal_type(Other.autos.through.objects)
         reveal_type(Other.autos.through._default_manager)
+
+        reveal_type(MyModel().auto_through)
+        reveal_type(MyModel().auto_through.through)
+        reveal_type(MyModel().other_again)
+        reveal_type(MyModel().other_again.through)
+        reveal_type(MyModel().custom_through)
+        reveal_type(MyModel().custom_through.through)
```

Maybe also check that the expected fields exist on through model?

```suggestion
        reveal_type(MyModel().auto_through.through)
        reveal_type(MyModel().auto_through.through.mymodel_id)
        reveal_type(MyModel().other_again)
        reveal_type(MyModel().other_again.through)
        reveal_type(MyModel().other_again.through)
        reveal_type(MyModel().other_again.through.mymodel_id)
        reveal_type(MyModel().custom_through)
        reveal_type(MyModel().custom_through.through)
        reveal_type(MyModel().custom_through.my_model_id)
```